src/main/kotlin/com/openlattice/assembler/processors/MaterializeEdgesProcessor.kt

```diff
@@ -47,8 +47,8 @@ data class MaterializeEdgesProcessor(val authorizedPrincipals: Set<Principal>)
             throw IllegalStateException("Encountered null assembly while trying to materialize edges for " +
                     "organization $organizationId.")
         } else {
-            acm?.materializeEdges(organizationId, assembly.materializedEntitySets.keys, authorizedPrincipals)
-                    ?: throw IllegalStateException(AssemblerConnectionManagerDependent.NOT_INITIALIZED)
+
+            //        ?: throw IllegalStateException(AssemblerConnectionManagerDependent.NOT_INITIALIZED)
```

Comment:
Not super important, but did you want to preserve these lines in a comment? Looks like one was commented and one was deleted.